README.md

```diff
@@ -35,384 +35,14 @@ while ab takes care of image builds only.
-```bash
-$ ansible-bender get-logs 2
-
-PLAY [Demonstration of ansible-bender functionality] *********************************
-
-TASK [Gathering Facts] ***************************************************************
-ok: [a-very-nice-image-20190302-160751828671-cont]
-
-TASK [Run a sample command] **********************************************************
-loaded from cache: 'cd27cfb71a161f3333232b97cc6b2a89354ff52de71bce9058c52cdf536735f9'
-skipping: [a-very-nice-image-20190302-160751828671-cont]
-
-TASK [Stat a file] *******************************************************************
-loaded from cache: '89ba4efc31358d688f035bf8159d900f1552314f0af6bf6c338b4897da593ccf'
-skipping: [a-very-nice-image-20190302-160751828671-cont]
-
-PLAY RECAP ***************************************************************************
-a-very-nice-image-20190302-160751828671-cont : ok=1    changed=0    unreachable=0    failed=0
-```
-
-
-### Locating built images with podman
-
-Once they are built, you can use them with podman right away:
-```bash
-$ podman images a-very-nice-image
-REPOSITORY                    TAG      IMAGE ID       CREATED         SIZE
-localhost/a-very-nice-image   latest   5202048d9a0e   2 minutes ago   83.5 MB
-```
-
-
-### Buildah configuration
-
-If you are familiar with podman and buildah, you know that you can
-[configure](https://github.com/containers/buildah/blob/master/docs/buildah.md#files)
-these tools. Ansible-bender doesn't change this configuration in any way so
-it's up to you how you set up buildah and podman. The same applies for running
-ansible-bender as root or not: buildah allows you to utilize rootless
-containers.
-
-My suggestion is to use the overlay storage backend. Vfs backend is slow and
-inefficient.
-
-
-### Caching mechanism
-
-Ansible bender has a caching mechanism. It is enabled by default. ab caches
-task results (=images). If a task content did not change and the base image is
-the same, the layer is loaded from cache instead of being processed again. This
-doesn't work correctly with tasks which process file: ab doesn't handle files
-yet.
-
-You are able to control caching in two ways:
-
- * disable it completely by running `ab build --no-cache`
- * or adding a tag to your task named `no-cache` — ab detects such tag and
-   will not try to load from cache
-
-
-### Layering mechanism
-
-When building your image by default, every task (except for setup) is being
-cached as an image layer. This may have bad consequences on storage and
-security: there may be things which you didn't want to have cached nor stored
-in a layer (certificates, package manager metadata, build artifacts).
-
-ab allows you to easily disable layering mechanism. All you need to do is to
-add a tag `stop-layering` to a task which will disable layering (and caching)
-for that task and all the following ones.
+## Contents
+* [Features](https://github.com/ansible-community/ansible-bender/docs/md_docs/features.md)
```

Comment:
I'd probably leave features in README so people can actually get interested in the project by the first look.

Reply:
In future, we can describe futures in detail in the documentation.